partcad-ide-vscode/bundled/tool/lsp_utils.py

```
@@ -80,7 +82,9 @@ class CustomIO(io.TextIOWrapper):
 
     name = None
 
-    def __init__(self, name, encoding="utf-8", newline=None, mirror=None, is_errors=False):
+    def __init__(
```

Ruff (0.8.2)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)